Sources/CSwiftScan/include/swiftscan_header.h

```diff
@@ -77,6 +77,18 @@ typedef struct {
 typedef struct swiftscan_scan_invocation_s *swiftscan_scan_invocation_t;
 typedef void *swiftscan_scanner_t;
 
+//=== CAS/Caching Specification -------------------------------------------===//
+typedef struct swiftscan_cas_s *swiftscan_cas_t;
+
+typedef enum {
+  SWIFTSCAN_OUTPUT_TYPE_OBJECT = 0,
+  SWIFTSCAN_OUTPUT_TYPE_SWIFTMODULE = 1,
+  SWIFTSCAN_OUTPUT_TYPE_SWIFTINTERFACE = 2,
+  SWIFTSCAN_OUTPUT_TYPE_SWIFTPRIAVEINTERFACE = 3,
```

Comment:
```suggestion
  SWIFTSCAN_OUTPUT_TYPE_SWIFTPRIVATEINTERFACE = 3,
```
Same typo in:
https://github.com/apple/swift/blob/6e47a3680871089934abff12a028b8ee2dc37d07/include/swift-c/DependencyScan/DependencyScan.h#L443C40-L443C40

Reply:
https://github.com/apple/swift/pull/67321